cmd/kubectl-gadget/version.go

```diff
@@ -58,7 +59,7 @@ var versionCmd = &cobra.Command{
 		// Get server version information
 		gadgetNamespaces, err := utils.GetRunningGadgetNamespaces()
 		if err != nil {
-			return fmt.Errorf("getting running Inspektor Gadget instances: %w", err)
+			fmt.Fprintf(os.Stderr, "Error: getting running Inspektor Gadget instances: %s\n", err)
```

Comment:
I wonder if the two `return fmt.Errorf` below (lines 70 and 76) should also be made non-fatal. What do you think?

Otherwise, the patch looks good to me.

Reply:
@alban yeah, that makes sense -- no reason not to print the client version in any case. I made the changes.